registrations/api.py

```diff
@@ -274,7 +299,12 @@ def filter_queryset(self, queryset):
                         _("Registration with id {pk} doesn't exist.").format(pk=pk)
                     )
 
-                if registration.publisher not in admin_organizations:
+                if not (
```

Comment:
Should we also allow superusers to have access?

Reply:
Allowed superusers to have access in https://github.com/City-of-Helsinki/linkedevents/pull/631/commits/2a60a351dca92e2849af5afd8ace752c381dda44